### DISCUSSION TITLE
Call to a member function prepare() on null

### DISCUSSION THREAD

**Discussion by m7vm7v** [spatie/laravel-multitenancy#456] [General]
Hi, I do understand that there are a few issues raised about that error but my configuration is slightly different and it's hit on another level so I wanted to hear your opinion on what could've caused that.

Most of the issues raised here are connected with BusQueue, but I do not use it in my application at all. Although it is connected to the Queue. I have tried the suggestions in these threads but no luck so far.

I have the `'queues_are_tenant_aware_by_default' => true,` in my config setup. The job itself does include the tenantId correctly. It seems that the first time after the job is executed it works fine, so restarting the queue fixes it for the first job.

The exact issue happens in this block for my application -
```
            Password::broker('users')->sendResetLink([
                'email' => $event->user->email // gets the correct email address every time so at this stage it picks the tenancy correctly 
            ]);
```

Weirdly the issue arises only in production. Can't replicate it locally or on the QA server.

Any ideas and suggestions are welcome. Many thanks

**Comment by masterix21** [comment]
Create a task to switch your default connection, then give me feedback.
